app/controllers/citizens/bank_transactions_controller.rb

```diff
@@ -3,7 +3,7 @@ class BankTransactionsController < ApplicationController
     include ApplicationFromSession
     before_action :authenticate_applicant!
 
-    def remove_transation_type
+    def remove_transaction_type
       bank_transaction.update! transaction_type: nil
       respond_to do |format|
         format.html do
```

Comment:
This code doesn't seem to work correctly.
The response of the AJAX call is the HTML one, not the JSON :
![image](https://user-images.githubusercontent.com/482806/54705898-b0055480-4b35-11e9-85a2-822dffe1f51b.png)

Replacing the `format.json` with `format.js` seems to fix the problem

Reply:
You would have to change the specs like so:
![image](https://user-images.githubusercontent.com/482806/54706723-7c2b2e80-4b37-11e9-9096-eeeafd123a4a.png)

Reply:
👍